docs/tutorials/clickstream/ksql-clickstream-demo.rst

```diff
@@ -0,0 +1,342 @@
+.. _ksql_clickstream:
+
+Clickstream Analysis Tutorial
```

Comment:
as above

Reply:
@bluemonk3y yes i have trimmed off tutorial for this one, but i think it should remain above on the higher level tutorial container topic.